### PR TITLE
misc(ui): image/asset tab tooltips, icon to rename board, getting started text

### PR DESCRIPTION
**Pull request by maryhipp**
## Summary

Misc UI:
* add tooltips to image/assets tab in gallery to explain what each is
* add an icon to boards on hover to more easily rename the board without knowing to right-click
* add info re prompt templates to getting started text on viewer

## Checklist

- [ ] _The PR has a short but descriptive title, suitable for a changelog_
- [ ] _Tests added / updated (if applicable)_
- [ ] _Documentation added / updated (if applicable)_

### CODE REVIEW

**Review by hipsterusername — APPROVED** [review]

File: invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx [hunk 1]
```diff
@@ -67,17 +77,28 @@ export const BoardEditableTitle = memo(({ board, isSelected }: Props) => {
...
+        </Text>
+        {isHovering && (
+          <IconButton
+            aria-label="edit name"
+            icon={<PiPencilBold />}
+            size="sm"
+            variant="ghost"
+            onClick={isEditing.setTrue}
+          />
+        )}
```
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:100]: Why is this needed? You can click the board title to edit it.

Comment (psychedelicious) [reply invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:100]: Also this introduces a race condition (ish) where the icon doesn't disappear.

https://github.com/user-attachments/assets/cf32e005-037e-424c-828e-2052da34147a

File: invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx [hunk 2]
```diff
@@ -67,17 +77,28 @@ export const BoardEditableTitle = memo(({ board, isSelected }: Props) => {
...
-        fontWeight="semibold"
-        userSelect="none"
-        color={isSelected ? 'base.100' : 'base.300'}
-        onDoubleClick={isEditing.setTrue}
-        cursor="text"
-        minW={16}
-      >
-        {localTitle}
-      </Text>
+      <Flex alignItems="center" gap={1} onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
```
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:80]: This gap is a bit too small.

image

Comment (psychedelicious) [reply invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:80]:
```suggestion
      <Flex alignItems="center" gap={3} onMouseOver={handleMouseOver} onMouseOut={handleMouseOut}>
```

File: invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx [hunk 3]
```diff
@@ -67,17 +77,28 @@ export const BoardEditableTitle = memo(({ board, isSelected }: Props) => {
...
+          {localTitle}
+        </Text>
+        {isHovering && (
+          <IconButton
+            aria-label="edit name"
+            icon={<PiPencilBold />}
+            size="sm"
+            variant="ghost"
+            onClick={isEditing.setTrue}
+          />
```
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:99]: `variant="link"` with `alignSelf="stretch"` looks better here.

File: invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx (hunk 1 above)
Comment (psychedelicious) [reply invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/BoardEditableTitle.tsx:100]: Ok, I think there's just a typo in the PR description - mentions right-click but it actually means double-click, all good. I'll tidy up the minor issues I've mentioned in a followup
